src/declaration.h

```diff
@@ -534,6 +535,7 @@ class FuncDeclaration : public Declaration
                                         // scopes from having the same name
     VarDeclaration *vthis;              // 'this' parameter (member and nested)
     VarDeclaration *v_arguments;        // '_arguments' parameter
+    Ojbc_FuncDeclaration objc;
```

Comment:
Has that branch anything to do with my pull request? Or are you just taking any opportunity to communicate :smiley: